### PR TITLE
Validate phone number input

### PR DESCRIPTION
**Pull request by groeneman**
Per Hillis's request, validate phone numbers using a simple regex

### CODE REVIEW

File: site/src/components/power-the-polls-form/power-the-polls-form.tsx [hunk 1]
```diff
@@ -163,10 +166,13 @@ export class PowerThePollsForm {
                </label>
 
                <label>
-                  Mobile phone
+                  Mobile phone<span class="ak-required-flag">*</span>
```
Comment (mojowen) [inline site/src/components/power-the-polls-form/power-the-polls-form.tsx]: I think we're ditching the `ak-required-flag` class - @nexussays can confirm

**Review by mojowen — APPROVED** [review]
Looks good - very small comment on the class name

File: site/src/components/power-the-polls-form/power-the-polls-form.tsx (hunk 1 above)
Comment (nexussays) [reply site/src/components/power-the-polls-form/power-the-polls-form.tsx]: Yes, holdover from the old design

Comment (groeneman) [reply site/src/components/power-the-polls-form/power-the-polls-form.tsx]: Removed!

**Pull request merged**